modules/instrument_builder/jsx/react.questions.js

```diff
@@ -916,7 +916,7 @@ class AddElement extends Component {
       if (minDate > maxDate && min !== '' && max !== '') {
         let temp = (this.state.error) ? this.state.error : {};
 
-        temp.dateOption = 'End year append before start year';
+        temp.dateOption = 'End year happened before start year';
```

Comment:
I think this needs to be rebased, this was a different PR.

Reply:
It was a previous PR that had already been merged. Have been trying to rebase or close it and issue a new PR but when I do so my PR's keep committing to the main branch while I made changes on my forked, I even used cherry-pick @driusan @ridz1208 what do you suggest I should do?

Reply:
I have done the rebase @ridz1208